config/routes.rb

```diff
@@ -8,6 +8,7 @@
   end
   get '/auth/logindotgov/callback' => 'users/omniauth#callback'
   get 'users/none' => 'users#none'
+  get 'env' => 'env#index'
```

Comment:
This is a nit, but could we change `get 'env'` to `get '/env'` for consistency.

Reply:
Updated in 692be98